Automattic/wp-calypso
# Full Site Editing: rename template parts to templates

#### Changes proposed in this Pull Request

Renames `wp_template_part` to `wp_template` and in general all occurrences of the word "part".

Fixes https://github.com/Automattic/wp-calypso/issues/35079

#### Testing instructions

1. Apply https://github.com/Automattic/themes/pull/1174 to update your Modern Business theme.
2. Delete previous Templates and Template Parts entries.
3. Apply this branch to Calypso.
4. Map this theme to your local Gutenberg Docker dev environment, in a similar fashion you do for FSE plugin: `{themes_repo_path}/themes/modern-business:/var/www/html/wp-content/themes/modern-business`
5. Delete `modern-business-fse-template-data` option if it was set on your site. Otherwise this would prevent data population here https://github.com/Automattic/themes/pull/1167/files#diff-8573df061382abec8228a31b5eb02880R84.
6. Activate Modern Business theme to trigger the activation hook.
7. Make sure that Templates are populated with correct content. You shouldn't see Template Parts entry anymore.
8. Make sure that everything still works as before.
9. Verify that the word "part" has been completely eradicated. :)
